src/tools/tidy/src/allowed_run_make_makefiles.txt

```diff
@@ -16,7 +16,9 @@ run-make/dump-ice-to-disk/Makefile
 run-make/emit-to-stdout/Makefile
 run-make/export-executable-symbols/Makefile
 run-make/extern-diff-internal-name/Makefile
-run-make/extern-flag-disambiguates/Makefile
+run-make/extern-fn-explicit-align/Makefile
```

Comment:
Perhaps you had some extra tests locally that got included here by `tidy`?

Reply:
rebasing to master showed this being added, maybe it should've been removed again in another commit but it didn't because that file already had a manually resolved conflict??

Reply:
regardless, it has been fixed.